oh_my_minishell/execute_pwd.c

```diff
@@ -0,0 +1,28 @@
+/* ************************************************************************** */
+/*                                                                            */
+/*                                                        :::      ::::::::   */
+/*   execute_pwd.c                                      :+:      :+:    :+:   */
+/*                                                    +:+ +:+         +:+     */
+/*   By: jikang <jikang@student.42seoul.kr>         +#+  +:+       +#+        */
+/*                                                +#+#+#+#+#+   +#+           */
+/*   Created: 2021/01/04 21:00:50 by jikang            #+#    #+#             */
+/*   Updated: 2021/01/06 19:28:43 by jikang           ###   ########.fr       */
+/*                                                                            */
+/* ************************************************************************** */
+
+#include "minishell.h"
+
+/*
+** https://man7.org/linux/man-pages/man2/getcwd.2.html
+** char *getcwd(char *buf, size_t size);
+** getcwd 는 buf 가 NULL 이면, 동적할당을 해준다. 그래서 free 필요.
+*/
+
+void execute_pwd(void)
+{
+	char *pwd;
+
+	pwd = getcwd(NULL, 1024);
```

Comment:
```suggestion
	if (!(pwd = getcwd(NULL, PATH_MAX)))
		ft_putendl_fd(strerror(errno), 2);
```
* PATH_MAX라는게 있어서 그걸 사용하는것도 괜찮을듯
* errno가 세팅되는 경우가 언젠지는 모르겠으나 있으니 저렇게 해주는게 좋을 것같음 에러메시지는 동일하지 않을탠데 그건 나중에 수정하면될듯 (나중에 errno를 처음에 초기화 해줘야할수도 있음)

Reply:
다음 커밋에서 수정 하였습니다, 감사합니다~